src/components/Input/types.ts

```diff
@@ -43,12 +46,8 @@ export const defaultProps = (props: InputPropTypes, className?: string) => {
       className || ""
     } ${props.className || ""}`,
     id: props.name,
-    name: props.name,
-    value: props.value,
-    disabled: props.disabled,
-    placeholder: props.placeholder,
-    style: props.style,
     onChange: (e: React.ChangeEvent<HTMLInputElement>) =>
       props.onChange ? props.onChange(e.target.value) : () => {},
+    ...props, // Default props
```

Comment:
Isn't it better to move `...props` to the top as it overrides the `onChange` property?

Reply:
yea, my bad. Updated the PR @Nandan-unni